content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx

```diff
@@ -19,17 +19,47 @@ We are continuously testing the Data Proxy with Cloudflare Workers. [Open an iss
 
 </TopBlock>
 
-
 ## Prerequisites
 
-- Free [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) database with an accessible URL
-  - e.g. `mongodb+srv://<user>:<password>@cluster.mongodb.net`.
-- Free [Cloudflare Worker](https://workers.cloudflare.com/) account.
-- Free [GitHub](https://github.com/) account.
-- Node.js & NPM installed.
-- Git installed.
+- Free [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) account
+- Free [Cloudflare Workers](https://workers.cloudflare.com/) account
+- Free [GitHub](https://github.com/) account
+- Node.js & NPM installed
+- Git installed
+
+## 1. Get a MongoDB Atlas connection string
+
+After you create a MongoDB Atlas database cluster, you can get its connection string. By default, that connection string does not include a database name.
+
+When you import your project in the Prisma Data Platform at [step 8](#8-import-your-project-into-the-prisma-data-platform), you must provide a connection string that includes a database name.
+
+You do not need to manually create the database in MongoDB Atlas even if it does not yet exist. The project creation workflow in the Prisma Data Platform will create the database.
+
+1. [Create a free shared cluster](https://www.mongodb.com/docs/atlas/tutorial/create-new-cluster/) in MongoDB Atlas.
+2. On the _Security Quickstart_ screen, complete the initial security configuration.  
+   1. Add a new user and save the password you assign to it. 
```

Comment:
Is it super-obvious how to add a new user? Typically,  in a procedure, we would write something like "To add a new User, enter the username in the ...and then click..."

Reply:
I think it is quite obvious. You have to do before you can complete the _Security Quickstart_ screen.

I also more than fully agree about the procedure style but I am also trying to quickly go over this without documenting too much of MongoDB beyond what we need for the tutorial.

<img width="1023" alt="05-prisma-mongodb-security-quickstart" src="https://user-images.githubusercontent.com/103522279/187300651-88fb6258-2860-43a7-94c4-beca1b731507.png">